backend/apps/users/forms.py

```
@@ -0,0 +1,17 @@
+
+
+
+class SignupForm(forms.Form):
+
+    email = forms.EmailField(label='Email', max_length=255)
+    password = forms.CharField(label='Password', widget=forms.PasswordInput)
+
+    def clean_email(self):
+        email = self.cleaned_data.get('email')
+        if not email.endswith('@usls.edu.ph'):
+            raise forms.ValidationError('Email must be a valid USLS email address')
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)